TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+### Add Newsletter Signup #7601
+
+- Start OBW and set up your browser console to monitor tracks. To do this, run `localStorage.setItem( 'debug', 'wc-admin:tracks' );`
```

Comment:
I think it should be `localStorage.setItem( 'debug', 'wc-admin:*' )` I wasn't able to get the debug messages show up with ``localStorage.setItem( 'debug', 'wc-admin:tracks' );``

Reply:
Good catch! ⚾

I updated it in 8c61c00

Reply:
And fixed a cop- paste error with that in 7114dc3 😅